robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -241,4 +241,26 @@ public void run() {
         .resume()
         .visible();
   }
+  
+  public ActivityController<T> configurationChange(final Application application, final Configuration newConfiguration) {
+    final T activity = get();
+    final Configuration currentConfig = activity.getResources().getConfiguration();
+    final int changedBits = currentConfig.diff(newConfiguration);
+    currentConfig.setTo(newConfiguration);
+    
+    // Can the activity handle itself ALL configuration changes?
+    if ((getActivityInfo(application).configChanges & changedBits) == changedBits) {
+      shadowMainLooper.runPaused(new Runnable() {
+        @Override
+        public void run() {
+          ReflectionHelpers.callInstanceMethod(Activity.class, component, "onConfigurationChanged",
+            ClassParameter.from(Configuration.class, newConfiguration));
+        }
+      });
+    } else {
+      Shadows.shadowOf(activity).recreateWithConfigChanges(changedBits);
```

Comment:
Why not inline the code here rather than add code to the ShadowActivity as now we have a new public non-android accessor on the Shadow. Would be good to avoid this.